components/Navbar.jsx

```diff
@@ -0,0 +1,79 @@
+import { Menu } from "@headlessui/react";
+import Image from "next/image";
+import Link from "next/link";
+import React, { useState } from "react";
+import Dropdown from "./Dropdown";
+import Fade from "./Fade";
+import { FaBars } from "react-icons/fa";
+import MobileNavbarDrawer from "./MobileNavbarDrawer";
+
+const textClasses = "text-xl font-semibold text-[#FF4D6F]";
+const optionClasses = "text-xl font-semibold text-white p-2";
+const optionsClasses = "absolute right-0 top-12 w-48 rounded-b-lg bg-[#FF4D6F] p-2";
+
+const DropdownOptions = ({ options }) => {
+  return (
+    <Menu.Items as="div" className={`absolute right-0 mt-2 origin-top-right divide-y ${optionsClasses}`}>
+      {options.map((option) => {
+        const link = option.toLowerCase().split(" ").join("-");
+        return (
+          <Menu.Item key={option} className={`flex justify-center items-center ${optionClasses}`}>
+            <a href={`/${link}</Menu.Item>`}>{option}</a>
+          </Menu.Item>
+        );
+      })}
+    </Menu.Items>
+  );
+};
+
+const Navbar = () => {
+  const [open, setOpen] = useState(false);
+
+  return (
+    <div className="flex flex-col relative">
+      <div className="rounded-b-2xl md:rounded-none flex justify-between items-center px-10 md:px-20 py-5 bg-white drop-shadow-lg z-10">
+        <FaBars className="md:hidden text-[#A689FF] cursor-pointer" onClick={() => setOpen(!open)} />
+
+        <Link href="/">
+          <Image className="cursor-pointer" src="/logo.png" alt="logo" width={44} height={44} />
+        </Link>
+
+        <div className="hidden md:flex md:items-center md:space-x-8">
+          <Dropdown
+            title="About"
+            titleClasses={textClasses}
+            optionsClasses={optionsClasses}
+            optionClasses={optionClasses}
+          >
+            <DropdownOptions options={["Who we are", "Projects"]} />
+          </Dropdown>
+          <Dropdown
+            title="News"
+            titleClasses={textClasses}
+            optionsClasses={optionsClasses}
+            optionClasses={optionClasses}
+          >
+            <DropdownOptions options={["Announcements", "Blog"]} />
+          </Dropdown>
+          <Link href="/partnerships">
+            <span className={textClasses}>Partnerships</span>
+          </Link>
+          <Link href="/contact">
+            <span className={textClasses}>Contact</span>
+          </Link>
+          <Link href="/join">
+            <span className={textClasses}>Join</span>
+          </Link>
```

Comment:
We can pretty easily turn this into a map function right?

```
const options = [
    ["/partnerships", "Partnerships"],
    ["/contact", "Contact"],
    ["/join", "Join"]
];

options.map(([href, title]) => {
    <Link href={href}>
        <span className={textClasses}>{title}</span>
    </Link>
});
```

Reply:
I just thought it would be weird to have half the tabs as mapped components and others as non-mapped. Might be better to just remain consistent.

Reply:
Sounds good